webapp/src/routes/Profile/ProfilePageLifebank.js

```diff
@@ -36,15 +48,168 @@ const useStyles = makeStyles((theme) => ({
+
+      if (data) setuserName(data.user[0].username)
+    }
+
+    if (!userName) getUsername()
+  })
+
+  const checkAvailableFields = () => {
+    let pendingFieldsObject = {}
+
+    if (!profile.email)
+      pendingFieldsObject = { ...pendingFieldsObject, email: false }
+
+    if (!profile.name)
+      pendingFieldsObject = { ...pendingFieldsObject, name: false }
+
+    if (!profile.telephones)
+      pendingFieldsObject = { ...pendingFieldsObject, telephones: false }
+
+    if (!profile.photos)
+      pendingFieldsObject = { ...pendingFieldsObject, photos: false }
+
+    if (!profile.logo_url)
+      pendingFieldsObject = { ...pendingFieldsObject, logo_url: false }
+
+    if (!profile.schedule)
+      pendingFieldsObject = { ...pendingFieldsObject, schedule: false }
+
+    if (!profile.location)
+      pendingFieldsObject = { ...pendingFieldsObject, location: false }
+
+    if (!profile.address)
+      pendingFieldsObject = { ...pendingFieldsObject, address: false }
+
+    if (!profile.about)
+      pendingFieldsObject = { ...pendingFieldsObject, about: false }
+
+    if (!profile.blood_urgency_level)
+      pendingFieldsObject = { ...pendingFieldsObject, blood_urgency_level: false }
+
+    if (Object.keys(pendingFieldsObject).length > 0)
+      setPendingFields(pendingFieldsObject)
+  }
+
+  useEffect(() => {
+    if (profile) {
+      checkAvailableFields()
+    }
+  }, [profile])
 
   return (
     <>
+      <div className={classes.divProgressProfile}>
+        {pendingFields && (
+          <Grid style={{ maxWidth: 500, margin: 'auto' }} item>
+            <Alert
+              action={
+                <Box
+                  display="flex"
+                  justifyContent="flex-end"
+                  alignItems="center"
+                >
+                  <LinkRouter
+                    style={{ textDecoration: 'none' }}
+                    to={{
+                      pathname: '/edit-profile',
+                      state: { isCompleting: true, userName: userName }
+                    }}
+                  >
+                    <Button
+                      color="secondary"
+                      className={classes.noCapitalize}
+                      classes={{
+                        root: classes.editBtn
+                      }}
+                    >
+                      Update
+                    </Button>
+                  </LinkRouter>
+                </Box>
+              }
+              className={classes.alert}
+              severity="info"
+            >
+              <Typography>Your profile is not complete </Typography>
+              <Box display="flex" alignItems="center">
+                <Box width="100%" mr={1}>
+                  <LinearProgress
+                    variant="determinate"
+                    color="secondary"
+                    className={classes.customizedLinearProgress}
+                    value={
+                      ((10 - Object.keys(pendingFields).length) * 100) / 10
```

Comment:
Why do you use 10?. I saw that lifebanks have 13 fields

Reply:
There are 11 editable fields (forget to add one) the rest will always be because they are created with the account, so do not consider them to measure the percentage of completeness of the profile.